eclide/RepositoryFrame.h

```diff
@@ -60,8 +60,6 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
     {
         T * pT = static_cast<T*>(this);
         ContextState state;
-        if (s.mods.size() + s.attrs.size() == 0)
-            return;
```

Comment:
@GordonSmith Inserting a Module file is allowed if clicked on a blue folder because that simply adds a new folder at the top level. It does not allow the insertion of a attribute file. I checked 6.2 and what it does and 6.4 with this change. They look identical and I can't see anything that it is allowing a user to do that would be "illegal".